### ISSUE TITLE
Refactor indicators endpoint

### ISSUE THREAD

**Issue by majkshkurti** [goat-community/goat#1740]
# Goal of this issue

- Convert /local-accessibility endpoint from GET to POST. As a rule of thumb, if there is a payload and payload contains json configuration (nested) the HTTP method should be POST
-  Move isochrone to the indicators section and rename it the same way as other indicators. 
(e.g /api/v1/indicators/isochrone)
